server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/variables/IndependentDataItemNode.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2021 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.core.model.tree.variables;
+
+import lombok.Getter;
+import lombok.ToString;
+import org.eclipse.lsp.cobol.core.messages.MessageTemplate;
+import org.eclipse.lsp.cobol.core.model.Locality;
+import org.eclipse.lsp.cobol.core.model.SyntaxError;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.eclipse.lsp.cobol.core.model.tree.variables.VariableDefinitionUtil.EMPTY_STRUCTURE_MSG;
+import static org.eclipse.lsp.cobol.core.model.tree.variables.VariableDefinitionUtil.LEVEL_77;
+
+/**
+ * This value class represents an independent element item COBOL variable, that has a level number
+ * 77. It should always have a PIC clause representing its type, and a VALUE clause that stores an
+ * explicitly defined value; both as Strings. They cannot produce a structure in any way.
+ */
+@Getter
+@ToString(callSuper = true)
+public class IndependentDataItemNode extends VariableWithLevelNode {
```

Comment:
I found this in the specs, I hope it helps:
```
data-item-description-entry
Independent items in the WORKING-STORAGE SECTION that bear no hierarchic relationship to one
another need not be grouped into records provided that they do not need to be further subdivided.
Instead, they are classified and defined as independent elementary items. Each is defined in a
separate data-item description entry that begins with either the level number 77 or 01. See Chapter
18, “DATA DIVISION--data description entry,” on page 179 for more information.
```

Reply:
> I found this in the specs, I hope it helps:
> 
> ```
> data-item-description-entry
> Independent items in the WORKING-STORAGE SECTION that bear no hierarchic relationship to one
> another need not be grouped into records provided that they do not need to be further subdivided.
> Instead, they are classified and defined as independent elementary items. Each is defined in a
> separate data-item description entry that begins with either the level number 77 or 01. See Chapter
> 18, “DATA DIVISION--data description entry,” on page 179 for more information.
> ```

That is why I wrote it is more general.  It can be used for both 01 or 77.  Better to use special name for 77.